tests/PrepareBodyMiddlewareTest.php

```diff
@@ -126,7 +126,7 @@ function (RequestInterface $request) {
             new Request('PUT', 'http://www.google.com', ['Expect' => 'Foo'], $bd),
             ['expect' => true]
         );
-        $this->assertInstanceOf('GuzzleHttp\Promise\FulfilledPromise', $p);
+        $this->assertInstanceOf('GuzzleHttp\Promise\PromiseInterface', $p);
```

Comment:
I forgot about that feature, but it would be nice to update the tests to use it at some point.